ironfish-cli/src/commands/deposit.ts

```diff
@@ -122,6 +122,12 @@ The memo will contain the graffiti "${graffiti}".
         this.log('Transaction aborted.')
         this.exit(0)
       }
+    } else {
+      this.log(`
+Your balance is ${confirmedBalance} and you are about to send ${displayAmount} plus a transaction fee of ${displayFee} to the Iron Fish deposit account.
+Your remaining balance after this transaction will be ${displayNewBalance}.
+The memo will contain the graffiti "${graffiti}".
+      `)
```

Comment:
no need to repeat this code in both if and else condition

Reply:
It isn't repeated. Look closer

Reply:
Repeating is fine, since we need to display the transaction transaction details before and after the transaction. But the wording is not. Just a wording mistake. I'll correct it

Reply:
It's because the if condition is when the confirm flag is not used, and the else when the confirm flag is used. The bug is that when the confirm flag is used, there is no output to show any details regarding the account balance

Reply:
i know, i'm just saying both conditions are printing 👇 

```
you are about to send ${displayAmount} plus a transaction fee of ${displayFee} to the Iron Fish deposit account.
Your remaining balance after this transaction will be ${displayNewBalance}.
The memo will contain the graffiti "${graffiti}".
```

Reply:
@wd021 I updated the code